articles/templates/about-template-output/about-template-output.md

```diff
@@ -0,0 +1,47 @@
+---
+uid: templates.about-template-output
+---
+# About Template Output
+
+Templates are used to produce generated content and to write them to files. Where these files are located can be determined by the `Output Configuration Designer`.
+
+![Output Config Template Output](images/output-config-template-output-side-by-side.png)
+_Example of Output Configuration Layouts_
+
+When Modules are installed that contain Text Templates, they will automatically install output destination items for those Templates called `Template Output` items. In the examples above you will notice that these `Template Output` items are located inside folders or SDK projects (for example, Visual Studio's SDK offers C# Projects as a way to organize files). You will also notice that these `Template Output` items are named according to their respective Template-Identifiers which are found in the Modules that are installed. This gives visibility to the user to know which Templates are present and where they are destined to produce file output during a Software Factory Execution.
+
+## Unassigned Template Output
+
+It may be possible that during Module development that a Template does not have a predefined `Role` (we will cover this in the next section) or the `Role` does not exist inside the `Output Configuration Designer`. This will cause the Template to be initially placed in the root level of your `Output Configuration Designer`.
+
+![Unassigned Template in Output Configuration Designer](images/output-config-vs-unassigned-template.png)
```

Comment:
We should change this to show a screenshot where the unassigned template is red:

![image](https://user-images.githubusercontent.com/1290262/168743011-d091289f-b6ff-40fe-8011-6c8c4490d403.png)